pkg/database/authorized_app.go

```diff
@@ -29,10 +29,14 @@ const (
 // AuthorizedApp represents an application that is authorized to verify
 // verification codes and perform token exchanges.
 // This is controlled via a generated API key.
+//
+// Admin Keys are able to issue diagnosis keys and are not able to perticipate
+// the verification protocol.
 type AuthorizedApp struct {
 	gorm.Model
-	Name   string `gorm:"type:varchar(100);unique_index"`
-	APIKey string `gorm:"type:varchar(100);unique_index"`
+	Name     string `gorm:"type:varchar(100);unique_index"`
+	APIKey   string `gorm:"type:varchar(100);unique_index"`
+	AdminKey bool   `gorm:"default:false"`
```

Comment:
Should we consider making this an enum (e.g. KeyType)? The boolean feels weird to me.

Reply:
changed to int / enum - right now I don't see the need for more than 2 key types, but I'm sure I'll be wrong tomorrow :)